### PR TITLE
Feature/tetris retrieve js

### PR DESCRIPTION
**Pull request by boylec1**
API added to retrieve game code files. Tested with Swagger (I don't really have a hook for where to put it in place to test in the actual website), as well as stepped through and checking return variables. It definitely grabs the GameCode.js file and returns it.

Also fixed the API pathing for leaderboard updating, and added the code to the other game files. Tetris and Snake will update leaderboards on BucStopGames now on game end. Not entirely sure why Pong wasn't doing this, but it was outside the scope of the task to debug that. I'm positive it's not the API code, it likely has to do with variable creation and/or where I set the call in the code, but setting it up for those two was mostly just courtesy since I was fixing the pathing anyways (which I needed to do to accomplish the actual JS code retrieval in the first place).

Sorry, hardly slept last night, writing a book.

TLDR: API for retrieving game code works, retrieves a string containing the entire .js code. API for leaderboards also works again. 

Screenshot of Swagger return for verification.
API game code retrieval success

### CODE REVIEW

**Comment by jboudou007** [comment]
Pretty much did the same, and we can now retrieve all files from all games including the thumbnail

**Review by jboudou007 — APPROVED** [review]
Tested. Looks good to me
